pkg/cvo/updatepayload.go

```diff
@@ -315,3 +317,15 @@ func findUpdateFromConfigVersion(config *configv1.ClusterVersion, version string
 	}
 	return configv1.Update{}, false
 }
+
+func (optr *Operator) clusterProfileRetriever() (string, error) {
+	configMap, err := optr.cmConfigLister.Get(internal.ClusterProfileConfigMap)
```

Comment:
This needs to check for [the `CLUSTER_PROFILE` environment variable][1] before looking at ConfigMaps or other locations.

[1]: https://github.com/openshift/enhancements/pull/200/files#diff-b0d3a9e089e5cf5a125a408f84f934abR67

Reply:
added!

Reply:
@guillaumerose Where is the env variable?

Reply:
Here https://github.com/openshift/cluster-version-operator/pull/404/files#diff-4f43e557624e0fc47bc4e31f4f236ab5R99

Reply:
I do not see the env viariable check before looking at ConfigMaps

Reply:
It's weird it doesn't appear here. If I browse the `Files changed` tab I can see it: https://github.com/openshift/cluster-version-operator/pull/404/files#diff-d540a41404f678a1c438f4c1e5b92a87R322-R325